projectmanagement/src/main/java/life/qbic/projectmanagement/application/ProjectCreatedEvent.java

```diff
@@ -0,0 +1,23 @@
+package life.qbic.projectmanagement.application;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.Instant;
+import java.util.UUID;
+import life.qbic.projectmanagement.domain.ProjectId;
+
+/**
+ * An integration event indicating successful project creation.
+ */
+public record ProjectCreatedEvent(UUID eventId, ProjectId projectId, Instant occurredOn) {
```

Comment:
Since this is a domain event, this belongs into the domain name space, not in the application name space

Reply:
This is not a domain event here but an integration event to integrate domains. Otherwise, we do not get any feedback from the application service. I think I might need to exchange the ProjectId with something that can exit the domain thou